releasenotes/notes/0.25/preset-pm-vf2-max-trials-958bb8a36fff472f.yaml

```diff
@@ -1,13 +1,13 @@
 ---
-upgrade:
+upgrade_transpiler:
   - |
     The maximum number of trials evaluated when searching for the best
```

Comment:
Didn't we backport this for 0.24.0 too? I think it's already in the published release notes for that.

Reply:
Hmm. We did backport to 0.24, but I don't see the release note there.

Reply:
I see it on the hosted docs: https://qiskit.org/documentation/release_notes.html#transpiler-upgrade-notes maybe it got moved around or something. I don't really remember what we did 3 months ago :p